api_tests/tests/sanity.ts

```diff
@@ -68,6 +68,66 @@ describe("Sanity", () => {
         })
     );
 
+    it(
+        "returns-invalid-body-for-bad-json-herc20-halbit",
+        oneActorTest(async ({ alice }) => {
+            const promise = alice.createHerc20Halbit({
+                // @ts-ignore
+                garbage: true,
```

Comment:
We already have some sanity test for the new endpoint (search `create-herc20-halbit-returns-invalid-body`). Please merge them all together.

Reply:
Cool, thanks. Will do.

Reply:
Done. Two additional patches pushed onto this PR.